script/legacy/2019-05-14-armagaddon/delete-from-gcs.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+## Affected period to delete is
+## 2019-05-04T11:00:00Z to 2019-05-11T11:00:00Z
+
+## Deletion in these buckets requires ops credentials.
+
+function delete_hours() {
+    local target_dir=$1
+
+    gsutil -m rm -r $target_dir/2019-05-04/11/
+    gsutil -m rm -r $target_dir/2019-05-04/12/
+    gsutil -m rm -r $target_dir/2019-05-04/13/
+    gsutil -m rm -r $target_dir/2019-05-04/14/
+    gsutil -m rm -r $target_dir/2019-05-04/15/
+    gsutil -m rm -r $target_dir/2019-05-04/16/
+    gsutil -m rm -r $target_dir/2019-05-04/17/
+    gsutil -m rm -r $target_dir/2019-05-04/18/
+    gsutil -m rm -r $target_dir/2019-05-04/19/
+    gsutil -m rm -r $target_dir/2019-05-04/20/
+    gsutil -m rm -r $target_dir/2019-05-04/21/
+    gsutil -m rm -r $target_dir/2019-05-04/22/
+    gsutil -m rm -r $target_dir/2019-05-04/23/
```

Comment:
if you generate the full list to delete you can pipe it to `gsutil -m rm -Ir`, for maximum efficiency